composer.json

```diff
@@ -138,7 +138,8 @@
     "codeception/phpunit-wrapper": "^7.0",
     "phpstan/phpstan": "^0.7",
     "symplify/easy-coding-standard-tester": "^4.4",
-    "symplify/changelog-linker": "^4.8"
+    "symplify/changelog-linker": "^4.8",
```

Comment:
should this be only in monorepo? would it make sense to include it in the packages and in `.travis` scripts? @vitek-rostislav